### ISSUE TITLE
Web & Networks Interest Group rechartering

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/strategy#239]
New charter proposal, reviewers please take note.

# Charter Review

[Charter: Web & Networks Interest Group](https://w3c.github.io/web-networks-charter/)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 
 - [x] Existing IG recharter
  https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2019%2F05%2Fweb-networks-ig-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fweb-networks-charter%2F

Communities suggested for outreach: 

Known or potential areas of concern?: 

Where would charter proponents like to see issues raised? (github preferred, email, ...)
* [github issues](https://github.com/w3c/web-networks-charter/issues)

Anything else we should think about as we review?

**Comment by himorin** [comment]
isn't it better to have dated link for patent policy in section 4?

**Comment by dontcallmedom** [comment]
Re dated link for the patent policy, I'm following the [model from the charter template](https://w3c.github.io/charter-drafts/charter-template.html#patentpolicy) which doesn't use a dated link.

**Comment by himorin** [comment]
> Re dated link for the patent policy, I'm following the [model from the charter template](https://w3c.github.io/charter-drafts/charter-template.html#patentpolicy) which doesn't use a dated link.

ah, sorry I seems to mis-remenber that part. let me rephrase as: isn't it better to include version of patent policy where it is mentioned? (as in the model `under the W3C Patent Policy (Version of 15 September 2020)`)

**Comment by dontcallmedom** [comment]
my understanding is that the template for IG charters wasn't updated with a dated mention since IGs only have disclosure obligations, not licensing ones, and so have been unaffected by the changes in the patent policy

**Comment by michael-n-cooper** [comment]
No comments from APA, thanks for the liaison statement! Over to @brewerj to complete accessibility horizontal review.

**Comment by himorin** [comment]
no comment/request from i18n.

**Comment by samuelweiler** [comment]
Removed HR checkboxes from the template, per strategy team discussion.

**Comment by samuelweiler** [comment]
Privacy and security reviews done.  Privacy issue at: https://github.com/w3c/web-networks-charter/issues/30
